### ISSUE TITLE
[Bug] Not able to use MemoryCard on no payload, please call load() error.

### ISSUE THREAD

**Issue by superchaoran** [wechaty/heroku-wechaty-getting-started#12]
```
chaoran$ node-ts src/main.ts
> heroku-wechaty-getting-started@1.0.0 start /Users/chaoran/Desktop/heroku-wechaty-getting-started
ts-node src/main.ts
15:19:05 INFO Wechaty <default> start() v0.26.1 is starting...
15:19:05 INFO startWeb startWeb() listening to http://localhost:8788
15:19:05 ERR PuppetPuppeteerBridge start() exception: Error: no payload, please call load() first.
15:19:05 ERR on-error onError(Error: Error: no payload, please call load() first.)
15:19:05 ERR PuppetPuppeteer initBridge() exception: no payload, please call load() first.
15:19:05 WARN PuppetPuppeteerBridge stop() page.close() exception: Error: Protocol error: Connection closed. Most likely the page has been closed.
15:19:05 ERR on-error onError(Error: Error: no payload, please call load() first.)
15:19:05 ERR PuppetPuppeteer start() exception: Error: no payload, please call load() first.
15:19:05 ERR on-error onError(Error: Error: no payload, please call load() first.)
15:19:05 WARN PuppetPuppeteerBridge stop() page.close() exception: Error: Protocol error: Connection closed. Most likely the page has been closed.
Error: no payload, please call load() first.
    at MemoryCard.<anonymous> (/Users/chaoran/Desktop/heroku-wechaty-getting-started/node_modules/memory-card/src/memory-card.ts:339:13)
    at Generator.next (<anonymous>)
    at /Users/chaoran/Desktop/heroku-wechaty-getting-started/node_modules/memory-card/dist/src/memory-card.js:11:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/chaoran/Desktop/heroku-wechaty-getting-started/node_modules/memory-card/dist/src/memory-card.js:7:12)
    at MemoryCard.get (/Users/chaoran/Desktop/heroku-wechaty-getting-started/node_modules/memory-card/dist/src/memory-card.js:256:16)
    at Bridge.<anonymous> (/Users/chaoran/Desktop/heroku-wechaty-getting-started/node_modules/wechaty-puppet-puppeteer/src/bridge.ts:202:51)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/chaoran/Desktop/heroku-wechaty-getting-started/node_modules/wechaty-puppet-puppeteer/dist/src/bridge.js:23:58)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
15:19:05 ERR Wechaty start() exception: no payload, please call load() first.
15:19:05 ERR on-error onError(Error: no payload, please call load() first.)
15:19:05 INFO Wechaty <default> stop() v0.26.1 is stoping ...
15:19:05 WARN PuppetPuppeteer stop() is called on a OFF puppet. await ready(off) and return.
15:19:05 INFO startFinis finisCallback(0, exit)
15:19:05 INFO startFinis finisCallback() bot is logout-ed
15:19:05 INFO Wechaty <default> stop() v0.26.1 is stoping ...
```

**Comment by huan** [comment]
This issue is because of the later version of Wechaty/MemoryCard requires to call `memoryCard.load()` to initialize the memory card instance.

The following diff will be able to fix it:

```diff
export function getWechaty (name: string): Wechaty {
  log.verbose('getWechaty', 'getWechaty(%s)', name)

  if (wechaty) {
    return wechaty
  }

  const memory = getMemory(name)
+  await memory.load()

  wechaty = new Wechaty({
    memory,
    name,
  })

  return wechaty
}
```

However, I'd like to enable the lazy loading for this scenario to compatible with this case.

**Comment by huan** [comment]
The latest master branch should fix this issue, with version 0.2.16 or above.

Please feel free to let me know if you have any other problems with Heroku, and thank you very much for your issue reporting, nice catch!

**Comment by superchaoran** [comment]
Resolved. Thanks!
